sdk/storage/Azure.Storage.DataMovement/tests/LocalFileStorageResourceTests.cs

```diff
@@ -21,7 +21,7 @@
 
 namespace Azure.Storage.DataMovement.Tests
 {
-    public class LocalFileStorageResourceTests : StorageTestBase<StorageTestEnvironment>
+    public class LocalFileStorageResourceTests : DataMovementTestBase
```

Comment:
same

Reply:
probably can talk about this in the above questions I had about removing the disposing local directory.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/35408